rfcs/text/0012-orchestrator-field-set.md

```diff
@@ -19,9 +19,79 @@ add a new schema rather than change existing material.
 
 ## Fields
 
-<!--
-Stage 1: Describe at a high level how this change affects fields. Which fieldsets will be impacted? How many fields overall? Are we primarily adding fields, removing fields, or changing existing fields? The goal here is to understand the fundamental technical implications and likely extent of these changes. ~2-5 sentences.
--->
+The proposed change adds six fields, as described below:
+
+```
+---
+- name: orchestrator
+  title: Orchestrator
+  group: 2
+  short: Fields relevant to container orchestrators.
+  description: >
+    Fields that describe the resources which container orchestrators manage or
+    act upon.
+  type: group
+  fields:
+    - name: cluster
+      level: extended
+      type: object
+      description: >
+        Orchestrator cluster details.
+
+    - name: cluster.name
+      level: extended
+      type: keyword
+      description: >
+        Name of the cluster.
+
+    - name: cluster.url
+      level: extended
+      type: keyword
+      description: >
+        URL of the cluster.
```

Comment:
I would suggest to remove this empty `cluster` object. It would be enough like this:
```suggestion
    - name: cluster.name
      level: extended
      type: keyword
      description: >
        Name of the cluster.

    - name: cluster.url
      level: extended
      type: keyword
      description: >
        URL of the cluster.
```

If we want to leave this to document the `cluster` object itself, we could place `name` as `url` as fields of this object.
```suggestion
    - name: cluster
      level: extended
      type: object
      description: >
        Orchestrator cluster details.
      fields:
      - name: name
        level: extended
        type: keyword
        description: >
          Name of the cluster.

      - name: url
        level: extended
        type: keyword
        description: >
          URL of the cluster.
```